src/steps/model/EnvironmentVariableKey.ts

```diff
@@ -1,5 +1,6 @@
 export default interface EnvironmentVariableKey {
   defaultValue?: string;
   name: string;
+  description: string;
```

Comment:
Description is nullable. Can you add the optional flag via the question mark like in default value.

description?: string;